minifi/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
```

Comment:
this file should go away completely / This is about the source LICENSE and thus whatever is in here that isn't in the NiFI/LICENSE should get added there and this file removed.